src/java.base/share/classes/java/io/DataInputStream.java

```diff
@@ -33,9 +33,10 @@
  * way. An application uses a data output stream to write data that
  * can later be read by a data input stream.
  * <p>
- * DataInputStream is not necessarily safe for multithreaded access.
- * Thread safety is optional and is the responsibility of users of
- * methods in this class.
+ * A DataInputStream is not safe for use by multiple concurrent
+ * threads. If a DataInputStream is to be used by more than one
+ * thread then access to the data output stream should be controlled
```

Comment:
"data output stream" should be "data input stream".

Reply:
I disagree. This is the sentence in context:

```
 * A data input stream lets an application read primitive Java data
 * types from an underlying input stream in a machine-independent
 * way. An application uses a data output stream to write data that
 * can later be read by a data input stream.
```